### ISSUE TITLE
SpawnKerbals not Removed from Roster after Recovery after Contract Failure

### ISSUE THREAD

**Issue by inigmatus** [jrossignol/ContractConfigurator#443]
Reported in https://github.com/inigmatus/GAP/issues/142

looks like an interesting loophole to gain crew members for free, albeit similarly named crew members ad infinitum.

Happens with SSI-Start: https://github.com/inigmatus/GAP/blob/master/SSI-Start.cfg

1. Pick up Inigma Kerman.
2. Fail contract by waiting for time to expire.
3. Recover Inigma Kerman.

Not even
```
BEHAVIOUR
{
    name = RemoveKerbal
    type = RemoveKerbal

    kerbal = @/kerbalsSSI.ElementAt(0)

}
```
prevents this.

**Comment by inigmatus** [comment]
Possibly related to #437 ?

**Comment by jrossignol** [comment]
Does Inigma Kerman need to be the active vessel for this?

**Comment by inigmatus** [comment]
No.

**Comment by jrossignol** [comment]
Sigh, related to #410 again.

**Comment by jrossignol** [comment]
No easy workaround here...  likely going to wait on KSP 1.1 to hopefully get this issue resolved.

**Comment by inigmatus** [comment]
thanks for looking into it. i guess my work around is to not hard fail contracts with SpawnKerbals where possible.

**Comment by inigmatus** [comment]
I can't seem to reproduce this. Closing https://github.com/inigmatus/GAP/issues/142 for now.

**Comment by jrossignol** [comment]
Fixed by KSP 1.1

**Comment by inigmatus** [comment]
Still reproducible. Try: https://www.dropbox.com/s/08aazslzliajmf3/Wright-AirplaneRide.cfg?dl=0

Simply EVA a Kerbal to fail the contract (max G bug with KSP, which is a separate issue *sigh*) and then hop back in your craft, and recover.  Tourists still appear on roster.  Not even RemoveKerbal behaviour can seem to remove them if I add it in.

**Comment by jrossignol** [comment]
Do me a favor and see if you can find the g force bug on the public
tracker, and if it's not there, raise a new one. Either way, ping me the
number.

**Comment by jrossignol** [comment]
Tracking under #564

**Comment by inigmatus** [comment]
fyi, KSP g force bug: http://bugs.kerbalspaceprogram.com/issues/4930

**Comment by jrossignol** [comment]
That one's closed out - can you raise a new one as per TriggerAu's request:

> Closing this report out for now. If you find it is still occuring in the latest version of KSP please open a new report (and this one can be linked to it.) For best results, the wiki contains really useful info for when creating a report http://bugs.kerbalspaceprogram.com/projects/ksp/wiki.
